spec/services/transaction_search/searcher_spec.rb

```diff
@@ -9,31 +9,31 @@
     let(:order) { create(:purchased_order, product: item) }
     let(:order_detail) { order.order_details.first }
     let(:account) { order.account }
-    let(:searcher) { described_class.new(TransactionSearch::AccountSearcher, TransactionSearch::DateRangeSearcher) }
+    let(:searchers) { [TransactionSearch::AccountSearcher, TransactionSearch::DateRangeSearcher] }
```

Comment:
In general, I think I prefer the instance to be initialized with the list of searchers, then you can search on that.

Reply:
👍